templates/catalog/_partials/product-cover-thumbnails.tpl

```diff
@@ -67,14 +67,6 @@
     </div>
   </div>
 
-
```

Comment:
@NeOMakinG 🤣🤣🤣

Reply:
"Will you remove it after we finish?" "Of course dude" 😁

Reply:
@Hlavtox thought the same while removing it, I was like "I'm not gonna let him do this anymore."

Reply:
🤣🤣